packages/angular/projects/ui-angular/src/lib/components/authenticator/components/confirm-sign-in/confirm-sign-in.component.html

```diff
@@ -6,7 +6,7 @@
     [disabled]="authenticator.isPending"
   >
     <amplify-slot name="confirm-sign-in-header" [context]="context">
-      <h3 class="amplify-heading">{{ headerText }}</h3>
+      <h3 class="amplify-heading amplify-heading--3">{{ headerText }}</h3>
```

Comment:
yes Vue authenticator needs this class as well, but instead of adding it manually like we are doing here in angular, Vue has a basic Heading component defined that it is using and so the heading class is being added there much like it is in React.